### DISCUSSION TITLE
TypeError: 'CTkSlider' object is not callable

### DISCUSSION THREAD

**Discussion by yusufbuyukbirer** [TomSchimansky/CustomTkinter#112] [General]
Hi, I'm trying to build a sort visualizer. 

I'm following the tutorial down below
https://python.plainenglish.io/build-a-sorting-algorithm-visualizer-in-python-f6f4afb1c98a

when I try to click sort button I'm getting this error. How can I fix this?

**Comment by TomSchimansky** [comment]
When you post no code, nobody will answer you, because how should one know what you are doing and where the error could come from? But I don't think this is a customtkinter problem so I will move this to discussions. Maybe you should consider asking such questions on StackOverflow.
